stock_picking_report_product_sticker/models/stock_picking.py

```diff
@@ -0,0 +1,35 @@
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    show_product_stickers = fields.Boolean(
+        help="Show Product Stickers on pickings of this type.",
+        compute="_compute_show_product_stickers",
+        store=True,
+    )
+    sticker_ids = fields.Many2many(
+        comodel_name="product.sticker",
+        string="Stickers",
+        compute="_compute_sticker_ids",
+        store=False,
+    )
+
+    @api.depends("picking_type_id.show_product_stickers")
+    def _compute_show_product_stickers(self):
+        for picking in self:
+            picking.show_product_stickers = (
+                picking.picking_type_id.show_product_stickers
+            )
+
+    @api.depends("show_product_stickers", "move_lines.product_id")
+    def _compute_sticker_ids(self):
+        self.update({"sticker_ids": None})
```

Comment:
IIRC this is the same for v15+ (it supports assigning to fields in batch):

```suggestion
        self.sticker_ids = False
```